src/main/java/be/seeseemelk/mockbukkit/entity/WindChargeMock.java

```diff
@@ -0,0 +1,47 @@
+package be.seeseemelk.mockbukkit.entity;
+
+import be.seeseemelk.mockbukkit.ServerMock;
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.WindCharge;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.UUID;
+
+/**
+ * Mock implementation of a {@link WindCharge}.
+ *
+ * @see AbstractWindChargeMock
+ */
+public class WindChargeMock extends AbstractWindChargeMock implements WindCharge
+{
+
+	/**
+	 * Constructs a new {@link WindChargeMock} on the provided {@link ServerMock} with a specified {@link UUID}.
+	 *
+	 * @param server The server to create the entity on.
+	 * @param uuid   The UUID of the entity.
+	 */
+	public WindChargeMock(@NotNull ServerMock server, @NotNull UUID uuid)
+	{
+		super(server, uuid);
+	}
+
+	@Override
+	public void explode()
+	{
+		if (!isInWorld())
+		{
+			return;
+		}
+
+		getWorld().createExplosion(this, getLocation(), 1.2F, false);
+		remove();
```

Comment:
Seems like this method does not trigger a EntityRemoveEvent. This is a separate issue though

Reply:
I have updated the method to call the EntityRemoveEvent.